framework/src/main/java/org/tron/core/net/message/handshake/HelloMessage.java

```diff
@@ -127,17 +129,23 @@ public String toString() {
             .append("lowestBlockNum: ").append(helloMessage.getLowestBlockNum()).append("\n");
 
     ByteString address = helloMessage.getAddress();
-    if (address != null && !address.isEmpty()) {
+    if (!address.isEmpty()) {
       builder.append("address:")
               .append(StringUtil.encode58Check(address.toByteArray())).append("\n");
     }
 
     ByteString signature = helloMessage.getSignature();
-    if (signature != null && !signature.isEmpty()) {
+    if (!signature.isEmpty()) {
```

Comment:
Is this field has some relationship with the codeVersion?

Reply:
> Is this field has some relationship with the codeVersion?

No, only code optimization.

Reply:
933756

Reply:
Ma ja sam na neki način i u teretani i ja Pa Pa

Reply:
https://t.me/notpixel/app?startapp=f7012815595_s654946https://t.me/notgames_bot/squads?startapp=cmVmPTcwMTI4MTU1OTU=

Reply:
Zakačite stavke ostave kako ne bi istekle nakon 1 časa

Reply:
https://t.me/notgames_bot/squads?startapp=cmVmPTcwMTI4MTU1OTU=0x1cfa5e643a3436d94732f65561f8c86e2ea910be

Reply:
Prevucite stavke iz međuspremnika da biste ih izbrisali

Reply:
🇦🇸🇦🇩🇦🇩Fg0x1cfa5e643a3436d94732f65561f8c86e2ea910beZakačite stavke ostave kako ne bi istekle nakon 1 časa